plugins/tests/service.rs

```diff
@@ -56,6 +56,27 @@ fn raw_idents() {
     }
 }
 
+#[test]
+fn service_with_cfg_rpc() {
+    #[tarpc::service]
+    trait Foo {
+        async fn foo(s: String, i: i32) -> (String, i32);
+        #[cfg(feature = "inexistant")]
```

Comment:
how about using `#[cfg(not(test))]` to make sure it actually gets compiled out? And then leave out the fn below?

```suggestion
        #[cfg(not(test))]
```

Reply:
Since `#[cfg(…)]` were not propagated the test was failing anyway, but you are right, leaving out the function in the impl is better.
Using `#[cfg(not(test))]` is a great idea!

I just fixed the commits according to your comment.